src/components/stop-watch/stop-watch.tsx

```diff
@@ -1,4 +1,4 @@
-import { Component, Prop } from "@stencil/core";
+import { h, Component, Prop } from "@stencil/core";
```

Comment:
```suggestion
import { Component, Prop } from "@stencil/core";
```

@peterennis we're not using this new import `h` further in the code.

Reply:
h is now required for Stencil.
https://github.com/ionic-team/stencil/blob/master/BREAKING_CHANGES.md